example/simple-server/README.md

```diff
@@ -0,0 +1,120 @@
+  * [Ziti Desktop Edge for Linux](https://openziti.github.io/ziti/clients/linux.html)
+
+# Requirements
+* go 1.19 or later
+* an OpenZiti network. If you do not have one, the [quickstart](https://openziti.github.io/ziti/quickstarts/quickstart-overview.html) works well
+* OpenZiti CLI to create services and identities on the OpenZiti Network
+* Have the appropriate Ziti Desktop Edge for your operating system
+  * [Ziti Desktop Edge for Windows](https://github.com/openziti/desktop-edge-win/releases)
+  * [Ziti Desktop Edge for Mac](https://apps.apple.com/app/id1460484572)
+  * [Ziti Desktop Edge for Linux](https://openziti.github.io/ziti/clients/linux.html)
+
+# Build the example
+```
+cd <repo-root-dir>/sdk-golang/example/simple-server/
+go build main.go
+```
+
+# Setup using the OpenZiti CLI
+These steps will configure the service using the OpenZiti CLI. At the end of these steps you will have created:
+* a service called `simpleService`
+* an identity to host (bind) the service
+* an identity to connect to (dial) the service
+* the service configs to connect the service to the overlay
+* the service policies required to authorize the identities for bind and dial
+
+Steps:
+1. log into Ziti. The host:port and username/password will vary depending on your network.
+
+       ziti edge login localhost:1280 -u admin -p admin
+1. Run this script to create everything you need.
+
+       echo Create the service configs
+       ziti edge create config simple.hostv1 host.v1 '{"protocol":"tcp", "address":"localhost","port":'8080'}'
+       ziti edge create config simple.interceptv1 intercept.v1 '{"protocols":["tcp"],"addresses":["simpleService.ziti"], "portRanges":[{"low":'8080', "high":'8080'}]}'
+
+       echo Create the service
+       ziti edge create service simpleService --configs "simple.hostv1,simple.interceptv1" --role-attributes simple-service
+       
+       echo Create two identities and enroll the server
+       ziti edge create identity user simple-client -a clients -o simple-client.jwt
+       ziti edge create identity device simple-server -a servers -o simple-server.jwt
+       ziti edge enroll --jwt simple-server.jwt
+       
+       echo Create service policies
+       ziti edge create service-policy simple-client-dial Dial --identity-roles '#clients' --service-roles '#simple-service'
+       ziti edge create service-policy simple-client-bind Bind --identity-roles '#servers' --service-roles '#simple-service'
+       
+       echo Create edge router policies
+       ziti edge create edge-router-policy simple-edge-router-policy --edge-router-roles '#all' --identity-roles '#clients,#servers'
+       ziti edge create service-edge-router-policy simple-service-edge-router-policy --edge-router-roles '#all' --service-roles '#simple-service'
+       
+       echo Run policy advisor to check
+       ziti edge policy-advisor services
+1. Run the server.
+
+       ./simple-server simple-server.json simpleService
+
+1. Enroll the client identity
```

Comment:
i'd just point them to the doc: https://openziti.github.io/ziti/identities/enrolling.html